developer-docs-site/docs/issues-and-workarounds.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Issues and Workarounds"
+slug: "issues-and-workarounds"
+---
+
+# Issues and Workarounds
+
+This page documents issues that frequently come up and the suggested workarounds. 
+
+:::tip Help keep this page up-to-date
+If you found an issue that is not on this page, submit a [GitHub Issue](https://github.com/aptos-labs/aptos-core/issues). Make sure to follow the issue format used in this document. 
+:::
+
+## Nodes
+
+### Terraform "Connection Refused" error
+
+#### Description
+
+When running terraform, the command errors out with a connection refused error message.
+
+  ```
+  Error: Get "http://localhost/api/v1/namespaces/aptos": dial tcp 127.0.0.1:80: connect: connection refused
+  ```
+
+#### Workaround
+
+This likely means that the state of the install is out of sync with the saved terraform state file located in the storage bucket (configured during `terraform init` statement).  This could happen if the cluster or other components were deleted outside of terraform, or if terraform had an error and did not finish.  Use the following commands to check the state.  Delete the state that is related to the error message.  You will likely need to run terraform destroy, clean up the environment, and run the terraform script again.  
+
+  ```
+  terraform state list
+
+  terraform state rm <state>
+  ```
+
+### Fullnode "NoAvailablePeers" error
+
+#### Description
+
+If your node cannot state sync, and the logs are showing "NoAvailablePeers", it's likely due to network congestion. 
+
+#### Workaround
+
+You can try add some extra upstream peers for your fullnode to state sync from. See the guide [Add upstream seed peers](/nodes/full-node/fullnode-source-code-or-docker#add-upstream-seed-peers).
+
+### Starting a node throws a YAML-parsing error
+
+#### Workaround
+
+YAML files are sensitive to formatting errors. Use a dedicated YAML editor or use a YAML syntax checker in your preferred editor to check if each line in the YAML file is indented correctly.
+
+### "Unable to fetch any peers to poll" error
+
+#### Description
+
+When starting a node with the `cargo run -p ...` command, you receive a "Unable to fetch any peers to poll" error. It looks like you have no peers on the available node testers. You have no output when running:
```

Comment:
I thought I heard Greg say we no longer recommend `cargo` use.

Reply:
I didn't spend any time editing it, just grabbed the troubleshooting sections scattered in multiple docs and put them here. Let's review it after we merge it. I am feeling a lack of time this week and prioritizing stuff fast.

Reply:
OK, let's leave for now.